### ISSUE TITLE
Issue: Freeze component advancing frames during pause

### ISSUE THREAD

**Issue by starwang666** [remotion-dev/remotion#4321]
Hello,

I'm currently working on a feature where I need the video to pause on frame 0 for 2 seconds, and after the 2-second delay, continue playing from frame 1. However, after using the Freeze component as shown below:
**<Freeze frame={0} active={(f) => f <= 60} />**
I noticed that after the 2-second pause, the video resumes from frame 60 instead of frame 1. This suggests that even though the frame is visually frozen, the video frames continue advancing in the background during the freeze.

Is there a way to adjust this so that the video resumes from frame 1 after the 2-second pause while using the Freeze component? Any advice on how to achieve this behavior would be appreciated.

Thank you!

**Comment by JonnyBurger** [comment]
Yes, wrap it in a `<Sequence>` where the `from` is dynamic as well

`<Sequence from={frame <= 60 ? 0 : 60}>`

I keep it open because we should probably make a documentation page and make a more intuitive example

**Comment by starwang666** [comment]
> 是的，将其包装在`<Sequence>`也是`from`动态的
> 
> `<Sequence from={frame <= 60 ? 0 : 60}>`
> 
> 我保持开放是因为我们应该制作一个文档页面并制作一个更直观的示例

Hello,

Thank you for your previous response, but I still have not resolved my issue. To clarify, I am using the Remotion Player component to play a video that contains two segments: Video 1 and Video 2. Each segment is structured with <Sequence><Freeze><OffthreadVideo /></Freeze></Sequence>.

My objective is to freeze Video 2 at frame 0 for 2 seconds using the following code:

jsx
复制代码
<Freeze frame={0} active={(f) => f <= 60} />
After the 2-second freeze, I want playback to continue from frame 1 of Video 2. However, if I change <Sequence from={frame <= 60 ? 0 : 60}>, it only adjusts the insertion point of Video 2 in the overall timeline, as mentioned in the documentation. This does not achieve the desired effect.

Could you please suggest any alternative solutions to implement this behavior?

Thank you!

![Snipaste_2024-09-24_12-15-07](https://github.com/user-attachments/assets/7a9a1fac-6205-4a4c-89d2-88193a6a8649)

**Comment by starwang666** [comment]
> 是的，将其包装`<Sequence>`也`from`动态化的
> 
> `<Sequence from={frame <= 60 ? 0 : 60}>`
> 
> 我保持开放是因为我们应该制作一个文档页面并制作一个更有趣的视觉效果

Hello,

Thank you for your previous response, but I still have not resolved my issue. To clarify, I am currently playing a complete video that consists of two segments: Video 1 and Video 2, using the Remotion Player component. Each segment is structured with <Sequence><Freeze><OffthreadVideo /></Freeze></Sequence>.

My objective is to freeze Video 2 at frame 0 for 2 seconds with the following code:
<Freeze frame={0} active={(f) => f <= 60} />

After the 2-second freeze, I want playback to continue from frame 1 of Video 2. However, if I change <Sequence from={frame <= 60 ? 0 : 60}>, it only adjusts the insertion point of Video 2 in the overall timeline, as noted in the documentation. This does not achieve the desired effect.

Could you please suggest any alternative solutions to implement this behavior?

Thank you!
![Snipaste_2024-09-24_12-15-07](https://github.com/user-attachments/assets/e3fa635f-8666-4e0d-93ef-cb98e279540e)

**Comment by starwang666** [comment]
> 是的，将其包装在`<Sequence>`也是`from`动态的
> 
> `<Sequence from={frame <= 60 ? 0 : 60}>`
> 
> 我保持开放是因为我们应该制作一个文档页面并制作一个更直观的示例

Hello,

Thank you for your previous response, but I still have not resolved my issue. To clarify, I am currently playing a complete video that consists of two segments: Video 1 and Video 2, using the Remotion Player component. Each segment is structured with `<Sequence><Freeze><OffthreadVideo /></Freeze></Sequence>`.

My objective is to freeze Video 2 at frame 0 for 2 seconds with the following code:
`<Freeze frame={0} active={(f) => f <= 60} />`

After the 2-second freeze, I want playback to continue from frame 1 of Video 2. However, if I change <Sequence from={frame <= 60 ? 0 : 60}>, it only adjusts the insertion point of Video 2 in the overall timeline, as noted in the documentation. This does not achieve the desired effect.

Could you please suggest any alternative solutions to implement this behavior?

Thank you!

![Snipaste_2024-09-24_12-15-07](https://github.com/user-attachments/assets/9c768533-5227-4e2e-9ec3-eb3daaf3a5cb)

**Comment by starwang666** [comment]
> Yes, wrap it in a `<Sequence>` where the `from` is dynamic as well
> 
> `<Sequence from={frame <= 60 ? 0 : 60}>`
> 
> I keep it open because we should probably make a documentation page and make a more intuitive example

Hello,

Thank you for your previous response, but I still have not resolved my issue. To clarify, I am currently playing a complete video that consists of two segments: Video 1 and Video 2, using the Remotion Player component. Each segment is structured with `<Sequence><Freeze><OffthreadVideo /></Freeze></Sequence>`.

My objective is to freeze Video 2 at frame 0 for 2 seconds with the following code:
`<Freeze frame={0} active={(f) => f <= 60} />`

After the 2-second freeze, I want playback to continue from frame 1 of Video 2. However, if I change `<Sequence from={frame <= 60 ? 0 : 60}>`, it only adjusts the insertion point of Video 2 in the overall timeline, as noted in the documentation. This does not achieve the desired effect.

Could you please suggest any alternative solutions to implement this behavior?

Thank you!
![Snipaste_2024-09-24_12-15-07](https://github.com/user-attachments/assets/51024368-9e54-4b04-8b77-df1ede438af7)

**Comment by starwang666** [comment]
> Yes, wrap it in a `<Sequence>` where the `from` is dynamic as well
> 
> `<Sequence from={frame <= 60 ? 0 : 60}>`
> 
> I keep it open because we should probably make a documentation page and make a more intuitive example

Hello,

Thank you for your previous response, but I still have not resolved my issue. To clarify, I am currently playing a complete video that consists of two segments: Video 1 and Video 2, using the Remotion Player component. Each segment is structured with `<Sequence><Freeze><OffthreadVideo /></Freeze></Sequence>`.

My objective is to freeze Video 2 at frame 0 for 2 seconds with the following code:
`<Freeze frame={0} active={(f) => f <= 60} />`

After the 2-second freeze, I want playback to continue from frame 1 of Video 2. However, if I change <Sequence from={frame <= 60 ? 0 : 60}>, it only adjusts the insertion point of Video 2 in the overall timeline, as shown in the screenshot in the documentation. This does not achieve the desired effect.

Could you please suggest any alternative solutions to implement this behavior?

Thank you!
![Snipaste_2024-09-24_12-15-07](https://github.com/user-attachments/assets/8a8c9b34-91bd-44bf-954f-8f20d80936ef)

**Comment by JonnyBurger** [comment]
Sorry, I instead meant

`<Sequence from={frame <= 60 ? 0 : -60}>`!

**Comment by starwang666** [comment]
> Sorry, I instead meant
> 
> `<Sequence from={frame <= 60 ? 0 : -60}>`!

Thank you for your response. However, adjusting the from property of the Sequence only modifies the entry point of the video sequence on the overall timeline. I would like to freeze the first frame of the video sequence for 2 seconds, and then after those 2 seconds, continue playing from the second frame of that sequence. Is there another method to achieve this? Thank you!

**Comment by starwang666** [comment]
> Sorry, I instead meant
> 
> `<Sequence from={frame <= 60 ? 0 : -60}>`!

Thanks for your answer — it really helped me solve the problem. I forgot to thank you earlier, so I'm saying it now. And also, you're welcome to visit China anytime, bro!

**Comment by JonnyBurger** [comment]
😊 Appreciate that, thanks a lot! 🇨🇳
